plugins/scheduler-docker-local/scheduler-deploy

```diff
@@ -130,7 +130,7 @@ scheduler-docker-local-scheduler-deploy() {
       # run checks first, then post-deploy hooks, which switches proxy traffic
       trap 'kill_new $cid $PROC_TYPE $CONTAINER_INDEX' INT TERM EXIT
       if [[ "$(is_app_proctype_checks_disabled "$APP" "$PROC_TYPE")" == "false" ]]; then
-        dokku_log_info1 "Attempting pre-flight checks"
+        dokku_log_info1 "Attempting pre-flight checks ($PROC_TYPE.$CONTAINER_INDEX)"
```

Comment:
The reason I moved from `log1` to `verbose` was because of the above log line. I wanted to signify that the sections below were all related, versus having them be completely separate.

Reply:
@michaelshobbs ^

Reply:
Taking a step back it's not clear even why these logging changes are in a bug fix PR. That minor point aside, it's still not clear to me what problem hiding those 2 specific log messages behind the verbose flag actually solves. Clarify that would probably help.

In absence of further clarity, however, those pauses above default to 10 seconds and are user configurable to be as high as one may desire. As part of providing a solid user experience we should be clear about any time we've paused processing of a command. Communicating what a program is doing in the background is a key principle of good UX. If there is, however, some question of the value of these log messages, perhaps we can modify them to provide more value?

Reply:
You're right, I can definitely separate out the two changes.

My original intention here was to reduce the amount of duplicative output. If a user is scaling a certain process up, then they'll get the same message about the CHECKS file over and over. Rather than do that, I wanted to only alert once so as to not to distract from the work being performed.

As I was refactoring, I also wanted to reduce the number of messages around attempts - or at least make them more clear. I noticed that we had a bug in the order in which attempts were performed, hence why that change tagged along in this PR.

Does that make sense?

Reply:
Got it. I wasn’t putting together the execution flow. Thank you for clarifying